front-end/src/services/auth.service.tsx

```diff
@@ -54,10 +56,23 @@ export const handleTokenChange = (token: string, currentUser: UserDetailsContext
 			} = tokenPayload;
 
 			currentUser.setUserDetailsContextState((prevState) => {
+				let addresses = '';
+				let defaultAddress = '';
+
+				if (NETWORK === 'kusama') {
```

Comment:
Thanks, that works.

we should use `network.KUSAMA` and `network.POLKADOT` with network coming from `src/global`. It'll make it easier for refactoring.
we should then eventually make a proper type out of it (like I just did for auth server). We can do that in another PR.